src/frontend/interfaces/action/build-action-click-handler.ts

```diff
@@ -8,22 +8,25 @@ import { actionHref } from './action-href'
 import { ActionJSON } from './action-json.interface'
 import { buildActionCallApiTrigger } from './build-action-api-call-trigger'
 import { TranslateFunctions } from '../../../utils'
+import { ModalData, ModalFunctions } from '../modal.interface'
```

Comment:
Please rename to: `ConfirmModalData`, `ConfirmModalFunctions` and `confirm-modal.interface`

We can have more modals later and `ModalXXX` is too generic